src/AspNet.Security.OAuth.Vkontakte/VkontakteAuthenticationDefaults.cs

```diff
@@ -52,6 +52,6 @@ public static class VkontakteAuthenticationDefaults
         /// <summary>
         /// Default API version.
         /// </summary>
-        public const string ApiVersion = "5.78";
+        public const string ApiVersion = "5.131";
```

Comment:
As this is a constant, technically this is a breaking change. Anything depending on this library wouldn't pick up the fix without explicit re-compilation.

I think for v5 it would be better to have the options class not use the constant, and instead assign this value directly.

Then in v6 we can make a breaking change and make this a property or field, rather than a constant.

Reply:
>Anything depending on this library wouldn't pick up the fix without explicit re-compilation.

I don't see that as problem. They need to recompile anyway to use new version. 

Scenarios that it won't be picked is:
1. ASP.NET project (Nuget version for provider updated) → Some other library (not recompiled) → Updated provider
2. ASP.NET project (not recompiled) → Provider (updated as binary directly on production)

I see both scenarios as extremely unlikely. Am I missing something?

Reply:
Sorry, it's early, so what I said isn't completely accurate.

The fix is fine, and not using the constant in the library itself would make no difference, but it wouldn't fix it for any library _using the constant_ as that would need recompiling against the fixed version to fix itself.

I agree that isn't likely (app depending on library X, which depends on this), but is a possible scenario. That said, there's no way to fix that retroactively without making a breaking change anyway. Of course any scenario in that predicament can just override the API version in code anyway.

I'll merge this once the test is fixed, then I'll merge this into the v6 branch and review all the usage of public constants and change them to fields where appropriate so the edge case for this doesn't persist going forwards.